README.md

```diff
@@ -29,7 +27,7 @@ For more information regarding the technology platforms, languages, APIs, and SD
 Esri welcomes contributions from anyone and everyone. Please do not contribute code you did not write yourself, unless you are certain you have the legal ability to do so. Also ensure all code contributed can be licensed under the Apache License 2.0. For more details please see our [guidelines for contributing](https://github.com/Esri/developer-support/blob/master/CONTRIBUTING.md).
 
 ## Licensing
-Copyright 2015 Esri
+©2015-2019 Esri
```

Comment:
Could we change 2019 to 2020 since it's the last day of the month?

Reply:
done in 720b3f7